packages/app-addresses/src/MemoByAccount.tsx

```diff
@@ -21,7 +21,7 @@ type State = {
 
 export default class Component extends React.PureComponent<Props, State> {
 
-  static getDerivedStateFromProps (props: Props, currentState: State) {
+  static getDerivedStateFromProps (props: any, currentState: State): State | null {
```

Comment:
Cool that you found the right place!  Could you try to change to this instead:
```suggestion
  static getDerivedStateFromProps (props: Props, currentState: State): State | null {
```

Reply:
Fixed. The Prop type needed to implement `RouteComponentProps`. I pushed a fix